pkgs/development/ruby-modules/bundler-env/default.nix

```diff
@@ -15,7 +21,13 @@
 , ...
 }@args:
 
+assert name != null || pName != null;
+
 let
+  dName =
```

Comment:
What does the "d" in `dName` mean here? After thinking about it for a bit, I figure "derivation"? If that's the case, it might be nitpicking, but `drvName` might be a bit clearer.